client/src/components/Channel.js

```diff
@@ -270,89 +269,70 @@ class Channel extends React.Component {
     this.node.scrollTop = this.node.scrollHeight + this.node.clientHeight;
   }
 
-  render() {
-    const theme = this.state.theme;
-    const channelMode = (<div className={"statusMessage"} style={theme}>{this.state.channelMode}</div>);
-
-    const controlsBar = (
-      <TransitionGroup
-        component="div"
-        transitionName="controlsAnimation"
-        transitionAppear={true}
-        transitionAppearTimeout={1000}
-        transitionEnterTimeout={0}
-        transitionLeaveTimeout={0}
-        >
-        <div className="Controls" key="controls">
-          <SendMessage onSendMessage={this.sendMessage.bind(this)} key="SendMessage" theme={theme} useEmojis={this.state.appSettings.useEmojis}/>
-          <Dropzone className="dropzone2" onDrop={this.onDrop.bind(this)} key="dropzone2">
-            <div className="icon flaticon-tool490" style={theme} key="icon"/>
-          </Dropzone>
-        </div>
-      </TransitionGroup>
-    );
-
-    const messages = this.state.messages.map((e) => {
-      return <Message
-                message={e.payload}
-                key={e.hash}
-                onDragEnter={this.onDragEnter.bind(this)}
-                highlightWords={this.state.username}
-                colorifyUsername={this.state.appSettings.colorifyUsernames}
-                useEmojis={this.state.appSettings.useEmojis}
-                style={{
-                  fontFamily: this.state.appSettings.useMonospaceFont ? this.state.appSettings.monospaceFont : this.state.appSettings.font,
-                  fontSize: this.state.appSettings.useMonospaceFont ? '0.9em' : '1.0em',
-                  fontWeight: this.state.appSettings.useMonospaceFont ? '100' : '300',
-                  padding: this.state.appSettings.spacing,
-                }}
-              />;
-    });
-
-    // let channelStateText = this.state.loading && this.state.loadingText ? this.state.loadingText : `Loading messages...`;
-    let channelStateText = this.state.loadingText ? this.state.loadingText : `???`;
-    if(this.state.reachedChannelStart && !this.state.loading)
-      channelStateText = `Beginning of #${this.state.channelName}`;
-
-    messages.unshift(<div className="firstMessage" key="firstMessage" onClick={this.loadOlderMessages.bind(this)}>{channelStateText}</div>);
-
-    const fileDrop = this.state.dragEnter ? (
-      <Dropzone
-        className="dropzone"
-        activeClassName="dropzoneActive"
-        disableClick={true}
-        onDrop={this.onDrop.bind(this)}
+  renderMessages() {
+    const { messages, username, channelName, loading, loadingText, reachedChannelStart, appSettings } = this.state;
+    const { colorifyUsernames, useEmojis, useMonospaceFont, font, monospaceFont, spacing } = appSettings;
+    const elements = messages.map(message => (
+      <Message
```

Comment:
In general, let's use parenthesis around function parameters to make it explicit that it's a function:
```javascript
const elements = messages.map((message) => ...)
```